src/tests/DbgShim.UnitTests/ConfigFiles/Windows/Debugger.Tests.Config.txt

```diff
@@ -118,7 +118,6 @@
   <BuildProjectFramework Condition="StartsWith('$(RuntimeFrameworkVersion)', '10')">net10.0</BuildProjectFramework>
```

Comment:
Could we put these in a common config file and import it into the config, or auto generate these with a script?  There seems to be duplication that we could remove here.

Reply:
I have thought about that, but just haven't done it yet.  It would need to be imported after the RuntimeFrameworkVersion is defined (it is defined locally)